### ISSUE TITLE
Accessibility Audit - Core pages, part 1

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#5001]
### Background
We need to complete an accessibility audit for Q1. This ticket tracks two of the pages/flows that need auditing.

### Action requested
Run Axe DevTools testing (automatic and manual) on the following pages/experiences in `stg`:

- [x] Dashboard
- [x] Conduct test (with at least 2 tests active)
- [x] AOE form

### Acceptance Criteria
Tests have been run on the above flows.

This ticket is only to track that the tests have been run, not that issues have been fixed! See the epic #4955 for details.

**Comment by emyl3** [comment]
Added AOE form! Found nothing on these pages.

**Comment by emyl3** [comment]
Oops... No wonder I was going through these so quickly... Just realized I that I wasn't actually running through all the "Guided tests" just by clicking that tab and saving the test... -- reopened the tickets and ACTUALLY going to run the guided tests this time haha... so sorry about that!!

@emmastephenson do we want to also run through the  "Remaining Testing" which links to https://cdc-devtools.dequecloud.com/coverage-page-state
![Screen Shot 2023-01-18 at 19.02.12.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/2c55ecb5-06da-465f-802d-fc803087ffb7)

Or is just  "Automatic Testing" and "Intelligent Guided Testing" fine?

**Comment by emmastephenson** [comment]
@emyl3 no worries! The interface isn't the most intuitive 😄 

I don't think we need to run through "Remaining Testing", just the automatic testing and guided testing should be fine. Thanks!

**Comment by emyl3** [comment]
ok now I'm actually done with this one haha -- thank you emma!
